src/openms/source/METADATA/ProteinIdentification.cpp

```diff
@@ -81,7 +81,8 @@ namespace OpenMS
     enzyme(UNKNOWN_ENZYME),
     missed_cleavages(0),
     peak_mass_tolerance(0.0),
-    precursor_tolerance(0.0)
+    precursor_tolerance(0.0),
+    digestion_enzyme()
```

Comment:
You would here use the constructor that contains one parameter for every member and not the default constructor.
e.g. digestion_enzyme("UKNOWN_ENZYME", ...